lib/schemas/eikjson.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
```

Comment:
publishing this file and adding an $id here should enable editors like vscode to do code completion (I think)